File: app/javascript/channels/test_channel.js
import consumer from './consumer'

// TestChannel is created only in lower envs for debugging
if (process.env.MG_DEBUG === 'true') {
    consumer.subscriptions.create('TestChannel', {
        connected() {
            this.send({ message: 'Client is live' })
        },

        received(data) {
            console.log(data)
        }
    })
}

File: app/channels/test_channel.rb
# frozen_string_literal: true

# Verify ActionCable connection in lower envs (TestChannel does not exist in production)
class TestChannel < ApplicationCable::Channel
  def subscribed
    stream_from "test"
  end

  def receive(data)
    puts data["message"]
    ActionCable.server.broadcast("test", "ActionCable is connected")
  end
end